README.md

```diff
@@ -23,37 +19,26 @@ YMMV; use in production at your own risk.
 
 # Requirements
 
-[Go 1.1](http://golang.org/doc/go1.1) or later is required.
+[Go 1.13](http://golang.org/doc/go1.13) or later is required.
 
 Tested against Neo4j 2.2.4 and Go 1.4.1.
```

Comment:
This line can also be updated.

Reply:
My logic for not updating it was: "I need 1.13 to build with modules, but the code itself is tested with 1.4.1".
Now reading it again the "go" part is kind of useless. We only need `Tested against Neo4j 2.2.4`